artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java

```diff
@@ -1328,6 +1330,17 @@ protected static final void waitForComponent(final ActiveMQComponent component,
       }
    }
 
+   protected static final void waitForBridges(final ActiveMQServer server, int connectedBridges) throws Exception {
+      waitForBridges(server.getClusterManager().getDefaultConnection(null), connectedBridges);
+   }
+
+   protected static final void waitForBridges(ClusterConnection clusterConnection, int connectedBridges) throws Exception {
+      Bridge[] bridges = clusterConnection.getBridges();
```

Comment:
This doesnt look used. Is it needed? Was a quick-exit check intended but missed?

Reply:
You are right I missed to remove that line, I'll create another PR to fix it.